engine/src/main/scala/cromwell/server/CromwellServer.scala

```diff
@@ -52,16 +52,18 @@ object CromwellServer {
 }
 
 class CromwellServerActor(config: Config) extends CromwellRootActor with CromwellApiService with SwaggerService {
-  implicit def executionContext = actorRefFactory.dispatcher
+  implicit def executionContext: ExecutionContextExecutor = actorRefFactory.dispatcher
 
   override val serverMode = true
   override val abortJobsOnTerminate = false
 
-  override def actorRefFactory = context
-  override def receive = handleTimeouts orElse runRoute(possibleRoutes)
+  override def actorRefFactory: ActorContext = context
+  override def receive: PartialFunction[Any, Unit] = handleTimeouts orElse runRoute(possibleRoutes)
 
-  val possibleRoutes = workflowRoutes.wrapped("api", config.as[Option[Boolean]]("api.routeUnwrapped").getOrElse(false)) ~ swaggerUiResourceRoute
-  val timeoutError = APIResponse.error(new TimeoutException("The server was not able to produce a timely response to your request.")).toJson.prettyPrint
+  val routeUnwrapped: Boolean = config.as[Option[Boolean]]("api.routeUnwrapped").getOrElse(false)
```

Comment:
We've had this debate before but I'll say my stance again. My beef with autoformatters for complex languages like Scala is that they tend to enforce a one size fits all formatting and I don't believe that's appropriate. For simplistic languages like Python, that's a different matter. I don't know this to be true but I've always suspected that's why projects like scalacheck are really "bad" in this way.